publisher-sdk/src/androidTest/java/com/criteo/publisher/CriteoBannerViewIntegrationTest.java

```diff
@@ -49,13 +50,22 @@ public void setup() throws Exception {
 
     runOnMainThreadAndWait(() -> {
       criteoBannerView = new CriteoBannerView(
-          InstrumentationRegistry.getContext(),
+          InstrumentationUtil.getApplication().getApplicationContext(),
           bannerAdUnit);
     });
 
     criteoBannerView.setCriteoBannerAdListener(criteoBannerAdListener);
   }
 
+
+  @Test
+  public void instantiate_GivenContext_InitializeCorrectly() throws Exception {
+    runOnMainThreadAndWait(() -> {
+      CriteoBannerView criteoBannerView = new CriteoBannerView(InstrumentationUtil.getApplication().getApplicationContext(), null);
+      assertThat(criteoBannerView.getContext()).isEqualTo(InstrumentationUtil.getApplication().getApplicationContext());
```

Comment:
`InstrumentationUtil...` is used twice here, (three times with the one in the setup method. What do you think about saving it in a local val or in a field ?